### ISSUE TITLE
MOV extension not being recognized as a Video when uploaded through the dropzone (drag/drop)

### ISSUE THREAD

**Issue by RBerntsen** [umbraco/Umbraco-CMS#17311]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.3.0

### Bug summary

When using the drag/drop functionality in the MediaPicker3 or directly in the Media section with a video of type MOV, the file is not being automatically identified as the media type "Video" but instead as the media type "File"

This is probably somewhat related to: [#16786]( https://github.com/umbraco/Umbraco-CMS/issues/16786)

### Specifics

I've used the MOV-file from here for testing:
https://file-examples.com/wp-content/storage/2018/04/file_example_MOV_480_700kB.mov

I have verified the mime-type of the file specified, which is "video/quicktime".

#### After uploading

![image](https://github.com/user-attachments/assets/103de635-b71c-4d62-ac40-108a779be8a7)

#### Possible place to start
I'm not completely sure where the mapping is done, but I can see input-upload-field.element.ts that's using the media/components/input-upload-field/utils.ts, that "mov" with aforementioned mime-type "video/quicktime" is specified here:

![image](https://github.com/user-attachments/assets/1f9798fd-b2df-4e4b-b0a7-3973b36f09ec)

https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/6bde8a27caafda53ae449e0d7f695c41d028a4ea/src/packages/media/media/components/input-upload-field/utils.ts#L382

Would probably make sense to start by verifying that the mapping is actually done correctly here?

### Steps to reproduce

- Install a clean version of Umbraco 14.3.0
- Download the MOV-file from [here](https://file-examples.com/wp-content/storage/2018/04/file_example_MOV_480_700kB.mov) - you can probably use any MOV-file, I've just tested with this one.
- Go to the media section
- Drag and drop the video into the dropzone (upload)
- Video will be uploaded
- Now check the media type that it has been created as (file type)

![ezgif-7-4f36e7e5d2](https://github.com/user-attachments/assets/16449d85-b3f5-4e03-a0b4-b4d643a23bdb)

### Expected result / actual result

When uploading a file of type mov with the mimetype "video/quicktime" I would expect the media type would be "video" and not "file".

**Comment by RBerntsen** [comment]
I just checked this against V15.1.1 and it still seems to be an issue. Now the video controls are displayed on the file though. Maybe something related to what was fixed in [#2116](https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/2116).

Here the controls are now shown:
![image](https://github.com/user-attachments/assets/00f29c35-e091-4805-8dff-b1bb5d5e1f25)

But still identified as a file type and not video:
![image](https://github.com/user-attachments/assets/050a040a-f03b-4892-bffd-c4f269e565df)

**Comment by AndyButland** [comment]
I've had a look into this one and it seems like the functionality is working as expected.  The mapping for what extensions are associated with the different media types are held in their associated data types, in this case the one called "Accepted file extensions".  By default, the `.mov` extension isn't in the list, but if you add it, a file with this extension will be associated with the "Video" media type.

**Comment by RBerntsen** [comment]
@AndyButland So it's not something we can configure, right?

**Comment by AndyButland** [comment]
Yes, you configure it on the data type called "Upload Video" - via the property "Accepted file extensions".

(Sorry, I re-read my message from above and realised I garbled it a bit - corrected, so hopefully that's clear now).
